### ISSUE TITLE
Remove the Jekyll BBE update workflow

### ISSUE THREAD

**Issue by praneesha** [ballerina-platform/ballerina-dev-website#4934]
## Description
Need to remove [1] from the `mater` branch once the new BBE design is updated on it. This is used in the Jekyll build and is not needed for the new BBE design.

This is there in both `dev` and `master` and once removed from master, the change will be automatically applied to the `dev` branch via the daily auto sync.

[1] https://github.com/ballerina-platform/ballerina-dev-website/blob/master/.github/workflows/update_ballerina_by_example_patch_update.yml

## Related website/documentation area
> Add/Uncomment the relevant area label out of the following. 

## Describe your task(s)
> A detailed description of the task.

## Related issue(s) (optional)
> Any related issues such as sub tasks and issues reported in other repositories (e.g., component repositories), similar problems, etc. 

## Suggested label(s) (optional)
> Optional comma-separated list of suggested labels. Non committers can’t assign labels to issues, and thereby, this will help issue creators who are not a committer to suggest possible labels.

## Suggested assignee(s) (optional)
> Optional comma-separated list of suggested team members who should attend the issue. Non committers can’t assign issues to assignees, and thereby, this will help issue creators who are not a committer to suggest possible assignees.

**Comment by YasithD** [comment]
The above [workflow](https://github.com/ballerina-platform/ballerina-dev-website/blob/master/.github/workflows/update_ballerina_by_example_patch_update.yml) is removed and replaced with the new [workflow](https://github.com/ballerina-platform/ballerina-dev-website/blob/website-revamp/.github/workflows/ballerina_by_example_auto_update.yml) to generate BBEs for the new `Next.js` website. Once the `website-revamp` branch is merged to master this will be removed automatically.

**Comment by praneesha** [comment]
Fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/5182.
